Build.fsx

```diff
@@ -69,7 +69,7 @@ Target "PatchAssemblyVersions" (fun _ ->
         buildVersion.fileVersion
         buildVersion.infoVersion
 
-    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.*"
+    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.fs"
```

Comment:
We might want to add this as a comment then? So we can remember to remove in the future? 😃